Client/ApiClient.cs

```diff
@@ -343,6 +355,18 @@ public ApiClient(String basePath = "https://apitest.cybersource.com")
                     RestClient.Proxy = Configuration.Proxy;
                 }            
 
+                // Adding Client Cert
+                if(Equals(bool.Parse(Configuration.MerchantConfigDictionaryObj["enableClientCert"]), true))

```

Comment:
This condition will not always work. 

- What happens if I have set `enableClientCert` to `true` but I am trying to use Payments API ? 
- What happens if I have set `enableClientCert` to `true` but I am trying to hit non-MutualAuth environment?

Reply:
It'll work. Sending client cert won't create an issue.